pkg/integration/tests/stash/stash.go

```diff
@@ -0,0 +1,32 @@
+package stash
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var Stash = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Stashing files",
+	ExtraCmdArgs: "",
+	Skip:         false,
+	SetupConfig:  func(config *config.AppConfig) {},
+	SetupRepo: func(shell *Shell) {
+		shell.EmptyCommit("initial commit")
+		shell.CreateFile("file", "content")
+		shell.GitAddAll()
+	},
+	Run: func(shell *Shell, input *Input, assert *Assert, keys config.KeybindingConfig) {
+		assert.StashCount(0)
+		assert.WorkingTreeFileCount(1)
+
+		input.PressKeys(keys.Files.ViewStashOptions)
+		assert.InMenu()
+
+		input.PressKeys("a")
+		input.Type("stash name")
```

Comment:
we should assert that when 'a' is pressed, a prompt appears. Here's an example from another file:
```go
input.PressKeys("a")
assert.InPrompt()
assert.MatchCurrentViewTitle(Equals("Enter a file name"))
input.Type("my file")
```